.github/workflows/ci.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches:
       - main
-      - v4main
+      - v3
```

Comment:
in reality we probably won't use it, but in theory that's the branch we would merge PRs into when updating v3 branch. So that should then trigger a CI run after PR merge. So without this line we would get CI run on PRs but not on the final merge commit